iceoryx_hoofs/include/iceoryx_hoofs/error_handling/error_handling.hpp

```diff
@@ -36,6 +36,7 @@ namespace iox
     error(POSH__RUNTIME_FACTORY_IS_NOT_SET) \
     error(POSH__RUNTIME_IS_CREATED_MULTIPLE_TIMES) \
     error(POSH__RUNTIME_PUBLISHER_PORT_NOT_UNIQUE) \
+    error(POSH__RUNTIME_SERVICE_DESCRIPTION_FORBIDDEN) \
```

Comment:
Maybe we should be more specific, the reason why it is forbidden is that it is used internally.

Reply:
I added the reason to the `LogWarn()` output.